.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -28,16 +28,18 @@ env:
   REGISTRY_URL: gcr.io
   PROJECT_ID: airqo-250220
 
-jobs:
+jobs:        
   image-tag:
     name: create image tag
     runs-on: ubuntu-latest
     outputs:
       build_id: ${{ steps.prep.outputs.build_id }} # build id
-      datetime: ${{ steps.prep.outputs.datetime }} # build date
+      datetime: ${{ steps.prep.outputs.datetime }} # build date      
+    # This job will only run if triggered by a permitted user
+    if: contains('Baalmart,danielogen', github.triggering_actor)
```

Comment:
[Check if workflow was triggered by specific users](https://github.com/airqo-platform/AirQo-frontend/pull/988/commits/04f5c846c37198b64346e248fb0a1dfd2bab8390)

Reply:
That’s right @Baalmart. I figured that they didn’t really prohibit non admins from running the action

Reply:
Cool, thanks for the clarification @123MwanjeMike